package.json

```diff
@@ -60,6 +60,7 @@
     ]
   },
   "dependencies": {
+    "dompurify": "^2.0.8",
```

Comment:
The size of this package is too big. May be better with a plugin?

If there is a remote link, it will detect whether there is a XSS filter plugin, if not, then load it asynchronously from unpkg.com or other CDN. And remind users that it is better to manually plugin the XSS filter.

Reply:
I agree the package size is big. but in our docs we have mentioned `~21KB gzipped` and I think we are around `~25KB` and this package is `~6KB` and after minimizing through terser, it might be around 2-3 KB more than `~25KB`.
Having plugin is good for us and it will be optional so I don't think existing users would be notified about the plugin. 
yes, we can put a console message but wouldn't it be better to ship it from the core ? 

I am in favor of having it in core or as a plugin but as its a quite security issue I was thinking of shipping initially with core and we can later move out to plugin may be in `v5`? cause already we overshoot the size limit.

Reply:
Regardless how we do it, the change should be in core to prevent the security hole.